data.rkt

```diff
@@ -753,7 +784,49 @@
          #:host (host host-id-manchester-makerspace)
          #:registration-url (u "")
          #:talks
-         '())
+         (list
+           (T #:speaker (speaker speaker-id-tigran-khandkar)
+              #:title "Item and Effect Creation in Minecraft"
+              #:description "Custom super-powerful items in Minecraft using command scripts and scoreboards."
+              #:artifacts '()
+              #:website #f
+              #:references '())
+
+           (T #:speaker (speaker speaker-id-kevin-kadow)
+              #:title "Manchester Makerspace's Custom Doorbell"
+              #:description "A networked and remote doorbell and surveillance system for the Manchester Makerspace."
+              #:artifacts '()
+              #:website #f
+              #:references '())
+
+           (T #:speaker (speaker speaker-id-lisa-???)
+              #:title "PiHole and self-hosted DNS"
+              #:description "Using PiHole to control DNS resolution, monitoring ads and tracking requests from apps as well as potentially blacklisting sites and providing fine-grained parental controls."
+              #:artifacts '()
+              #:website #f
+              #:references '())
+
+           (T #:speaker (speaker speaker-id-kyle-roucis)
+              #:title "Unreal's Blueprint visual scripting"
+              #:description "Visual scripting's sordid history and Epic's bizarre but seemingly successful creation and adoption of its Blueprints visual scripting system."
+              #:artifacts '()
+              #:website #f
+              #:references '())
+
+           (T #:speaker (speaker speaker-id-jack-jutzi)
+              #:title "Rust Web Service from Scratch"
+              #:description "A learning email newsletter and web service written from scratch in the Rust programming language."
+              #:artifacts (list (Link #f (u "https://github.com/jsjutzi/rust-zero-backend")))
+              #:website #f
+              #:references '())
+
+           (T #:speaker (speaker speaker-id-daniel-krol)
+              #:title "Self-hosted Navigation Maps."
+              #:description "Using Sandstorm and other distributed technologies to host your own Google Maps replacement with social features and more!"
+              #:artifacts '()
+              #:website #f
```

Comment:
@orblivion what should the website for this project be?

Reply:
Also, any links to source artifacts?

Reply:
By "source artifacts" do you mean "releases" on Github, or similar? I have nothing like that. But here's the source:

https://github.com/orblivion/sandstorm-share-a-map

It's also the best choice for the project's website, as it links to the app on the Sandstorm app market (still as an "experimental" app).

Reply:
> By "source artifacts" do you mean "releases" on Github, or similar? I have nothing like that. But here's the source:
> 
> https://github.com/orblivion/sandstorm-share-a-map
> 

I just meant the source itself being an artifact related the presentation, so yes - this is great. Some other things could slides or whatever supporting materials. Anything you made is an artifact, in that sense.

> It's also the best choice for the project's website, as it links to the app on the Sandstorm app market (still as an "experimental" app).

:+1: